lib/public/utilities/formatting/formatEmail.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+import { h } from '/js/src/index.js';
+
+/**
+ * Display the given email in a link component
+ *
+ * @param {string} email the email to display
+ * @return {Component} the email component
+ */
+export const formatEmail = (email) => email ? h('a', { href: `mailto:${email}` }, email.trim()) : '-';
```

Comment:
Would it not be better to perhaps use `trim` in the begining. before the ternary check and `mailto`?

Reply:
You are right